dspace/config/registries/schema-publicationIssue-types.xml

```diff
@@ -27,5 +27,149 @@
             delimited by commas.
         </scope_note>
     </dc-type>
+    
+    <!-- OpenAIRE4 Guidelines -->
+    <!-- 6.2.2. Attribute relatedIdentifierType (M) -->
+    <!-- specific type required by OpenAIRE4 Guidelines -->
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>ark</qualifier>
+        <scope_note>Archival Resource Key</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>arxiv</qualifier>
+        <scope_note>arXiv.org identifier</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>bibcode</qualifier>
+        <scope_note>Astrophysics Data System bibliographic codes; bibcodes can be resolved via 
+            http://adsabs.harvard.edu/abs/bibcode.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>doi</qualifier>
+        <scope_note>Digital Object Identifier</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>ean13</qualifier>
+        <scope_note>European Article Number, now renamed International Article Number, but retaining 
+            the original acronym, is a 13-digit barcoding standard which is a superset of the original 
+            12-digit Universal Product Code (UPC) system.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>eissn</qualifier>
+        <scope_note>International Standard Serial Number (electronic Version)</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>handle</qualifier>
+        <scope_note>Handle System Identifier</scope_note>
```

Comment:
Question for today's Entities meeting:  Should we create a new, separate field for Handles?  Currently DSpace is storing Handles in `dc.identifier.uri` (alongside DOIs or other identifiers).  Or, is this field meant for *externally created* Handles (e.g. if a publisher has created a Handle)?